README.md

```diff
@@ -61,17 +107,20 @@ To get this version of `ghcid` building you need to provide a specific version o
 
 ```
   extra = pkgs.haskellPackages.callCabal2nix "extra" (sources.extra) {};
-in
-rec
+
+  shell = myHaskellPackages.shellFor {
+    packages = p: [
 ```
 
 Then add it to the end of your `callCabal2nix` call:
 
 ```
       (pkgs.haskell.lib.justStaticExecutables
-          (pkgs.haskellPackages.callCabal2nix "ghcid" (sources.ghcid) {inherit extra;}))
+          (callCabal2nix "ghcid" (sources.ghcid) {inherit extra;}))
 ```
 
+Note: I am building `ghcid` with `haskellPackages` not `myHaskellPackages`. If the tool fails to build you might want to either use a different package set or modify one yourself so the tool has the right dependencies.
```

Comment:
I like this! Maybe we should mention the reason, as "to increase the odds that we'll fetch it from the binary cache.".

Reply:
Good idea, I'll add this.

Reply:
Since we are building `ghcid` ourselves there isn't a cache hit benefit we will get from using `haskellPackages`. The pre-compiled versions of the libraries in `haskellPackages` might be cached (think .hi and haddock files) and we will benefit from pulling those from a binary cache. In fact that is the only downside of using `fastHaskellPackages`, you don't get cached versions. However, we have to build `myHaskellPackages` anyways, so unless we add a new version of a library to `myHaskellPackages` that `ghcid` needs and our project does not need, we won't get any speed benefits.